fearless/Modules/Staking/StakingMain/ViewModel/StakingStateViewModelFactory+Alerts.swift

```diff
@@ -44,7 +58,21 @@ extension StakingStateViewModelFactory {
     func stakingAlertsForValidatorState(_ state: ValidatorState) -> [StakingAlert] {
         switch state.status {
         case .active:
-            return []
+            guard
```

Comment:
copypaste(( maybe define ledgerInfo in commonData?

Reply:
We can move this into separate function because it depends on era and ledger. This info available in nominator, validator and bonded states.